docs/UserGuide.md

```diff
@@ -251,6 +251,122 @@ Examples:
 * `deletetask 2` deletes the task with index 2 from the task list.
 
 
+### Grading an assignment: `grade`
+Grades a student’s assignment.
+
+Format: `grade n/STUDENT_NAME a/ASSIGNMENT g/GRADE`
+
+Parameters:
+* STUDENT_NAME The name of the student.
+* ASSIGNMENT The name of the assignment.
+* GRADE The grade of the student.
+
+Examples:
+* `grade n/Rayson a/Functional Expressionism g/1300` grades Rayson’s Functional Expressionism assignment as 1300.
+* `grade n/Wesley a/Rune Reading g/600` grades Wesley’s Rune Reading assignment as 600.
+
+
+### Viewing an assignment grade: `viewgrade`
+Shows a student’s assignment grade.
+
+Format: `viewgrade n/STUDENT_NAME a/ASSIGNMENT`
+
+Parameters:
+* STUDENT_NAME The name of the student.
+* ASSIGNMENT The name of the assignment.
+
+Examples:
+* `viewgrade n/Rayson a/Functional Expressionism` shows Rayson’s Functional Expressionism grade.
+* `viewgrade n/Wesley a/Rune Reading` shows Wesley’s Rune Reading grade.
+
+
+### Editing an assignment grade: `editgrade`
+Edits a student’s assignment grade.
+
+Format: `editgrade n/STUDENT_NAME a/ASSIGNMENT g/GRADE`
+
+Parameters:
+* STUDENT_NAME The name of the student.
+* ASSIGNMENT The name of the assignment.
+* GRADE The score of the student.
+
+Examples:
+* `editgrade n/Rayson a/Functional Expressionism g/1200` edits the grade of Rayson’s Functional Expressionism assignment to 1200.
+* `editgrade n/Wesley a/Rune Reading g/1000` edits the grade of Wesley’s Rune Reading assignment to 1000.
+
+
+### Deleting an assignment grade: `deletegrade`
+Deletes a student’s assignment grade.
+
+Format: `deletegrade n/STUDENT_NAME a/ASSIGNMENT`
+
+Parameters:
+* STUDENT_NAME The name of the student.
+* ASSIGNMENT The name of the assignment.
+
+Examples:
+* `deletegrade n/Rayson a/Functional Expressionism` deletes Rayson’s Functional Expressionism grade.
+* `deletegrade n/Wesley a/Rune Reading` deletes Wesley’s Rune Reading grade.
+
+
+### Commenting on an assignment: `comment`
```

Comment:
Might want to add another verb infront e.g addcomment, since its a getter setter pair that you have here
(addcomment, viewcomment)